page-components/tables.md

```diff
@@ -4,6 +4,1217 @@ title:  Tables
 category: Page components
 ---
 
+- [Use cases](#use-cases)
+- [Behavior](#behavior)
+- [Content guidelines](#content-guidelines)
+- [Style](#style)
+- [Code](#code)
```

Comment:
Do you need a code heading? There's nothing after it, and we link to the code throughout the rest of the page.